Silcence the build

koppor opened the pull request:
By switching to CircleCI, it is possible to get a really shrunk output.

See https://circleci.com/gh/OpenTOSCA/winery/20 for the small output.

Travis does not pass the `-q` parameter everywhere - and I find CircleCI more easy to configure.

![grafik](https://user-images.githubusercontent.com/1366654/41710183-51d8f2ac-7545-11e8-92fe-ef11b5045e81.png)

When this is approved, I will file a bug at https://bugs.eclipse.org to enable CircleCI for eclipse/winery

lharzenetter commented on docs/dev/index.md:
```diff
@@ -3,18 +3,18 @@
     - IntelliJ Ultimate setup: [config/IntelliJ IDEA/README.md](config/IntelliJ%20IDEA/)
     - Eclipse setup: [config/Eclipse/README.md](config/Eclipse/)
 - [Encoding](Encoding.md) - information about how percent-encoding is used at Winery
+- [Handling of Properties](property-handling.md)
```
Please edit the license also

lharzenetter commented on org.eclipse.winery.bpmn4tosca.converter.tobpel/src/test/resources/logback-test.xml:
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (c) 2017-2018 Contributors to the Eclipse Foundation
```
I guess this file was created, therefore please remove 2017

koppor replied:
I added ADR-0025

lharzenetter commented on org.eclipse.winery.common/src/test/resources/logback-test.xml:
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (c) 2017-2018 Contributors to the Eclipse Foundation
```
I guess this file was created, therefore please remove 2017

lharzenetter commented on org.eclipse.winery.compliance/src/test/resources/logback-test.xml:
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (c) 2017-2018 Contributors to the Eclipse Foundation
```
I guess this file was created, therefore please remove 2017

lharzenetter commented on org.eclipse.winery.generators.ia/src/test/resources/logback-test.xml:
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (c) 2017-2018 Contributors to the Eclipse Foundation
```
I guess this file was created, therefore please remove 2017

lharzenetter commented on org.eclipse.winery.model.bpmn4tosca/src/test/resources/logback-test.xml:
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (c) 2017-2018 Contributors to the Eclipse Foundation
```
I guess this file was created, therefore please remove 2017

lharzenetter commented on org.eclipse.winery.model.csar.toscametafile/src/test/resources/logback-test.xml:
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (c) 2017-2018 Contributors to the Eclipse Foundation
```
I guess this file was created, therefore please remove 2017

lharzenetter commented on org.eclipse.winery.model.selfservice/src/test/resources/logback-test.xml:
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (c) 2017-2018 Contributors to the Eclipse Foundation
```
I guess this file was created, therefore please remove 2017

lharzenetter commented on org.eclipse.winery.model.tosca.yaml/src/test/resources/logback-test.xml:
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (c) 2017-2018 Contributors to the Eclipse Foundation
```
I guess this file was created, therefore please remove 2017

lharzenetter commented on org.eclipse.winery.model.tosca/src/test/resources/logback-test.xml:
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (c) 2017-2018 Contributors to the Eclipse Foundation
```
I guess this file was created, therefore please remove 2017

lharzenetter commented on org.eclipse.winery.repository.client/src/test/resources/logback-test.xml:
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (c) 2017-2018 Contributors to the Eclipse Foundation
```
I guess this file was created, therefore please remove 2017

lharzenetter commented on org.eclipse.winery.repository.configuration/src/test/resources/logback-test.xml:
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (c) 2017-2018 Contributors to the Eclipse Foundation
```
I guess this file was created, therefore please remove 2017

lharzenetter commented on org.eclipse.winery.topologymodeler/src/test/resources/logback-test.xml:
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (c) 2017-2018 Contributors to the Eclipse Foundation
```
I guess this file was created, therefore please remove 2017

lharzenetter commented on org.eclipse.winery.yaml.common/src/test/resources/logback-test.xml:
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright (c) 2017-2018 Contributors to the Eclipse Foundation
```
I guess this file was created, therefore please remove 2017

koppor commented:
I added [ADR-0024](https://github.com/eclipse/winery/pull/279/files#diff-914a69bd858cc92a036c1be04e40c9a7) regarding CircleCI.

Regarding `logback-test.xml`, I added [ADR-0025](https://github.com/eclipse/winery/pull/279/files#diff-b6ebafda951e99f7fa2be5b707246cab).

koppor commented:
Bug to enable CircleCI for github.com/eclipse/winery: https://bugs.eclipse.org/bugs/show_bug.cgi?id=536180

lharzenetter commented:
As you can reed at the [reques](https://bugs.eclipse.org/bugs/show_bug.cgi?id=536180) CircleCI  will not be allowed for Winery. Therefore you need to silence the build on travis, or configure Jenkins correctly.

koppor commented:
Switched back to TravisCI and looks good: https://travis-ci.org/OpenTOSCA/winery/builds/397263324

lharzenetter commented on docs/adr/0024-use-travis-for-continuous-integration.md:
```diff
@@ -0,0 +1,44 @@
+
+## Decision Outcome
+
+Chosen option: "CircleCI", because
```
Please adjust the ADR and reference https://bugs.eclipse.org/bugs/show_bug.cgi?id=536180
